mono/metadata/appdomain.c

```diff
@@ -1324,6 +1302,24 @@ ves_icall_System_AppDomain_GetAssemblies (MonoAppDomainHandle ad, MonoBoolean re
 	return res;
 }
 
+MonoArrayHandle
+ves_icall_System_AppDomain_GetAssemblies (MonoAppDomainHandle ad, MonoBoolean refonly, MonoError *error)
+{
+	error_init (error);
+	MonoDomain *domain = MONO_HANDLE_GETVAL (ad, data);
+	return get_assembly_array_from_domain (domain, refonly, error);
+}
+
+#ifdef ENABLE_NETCORE
+MonoArrayHandle
+ves_icall_System_Runtime_Loader_AssemblyLoadContext_InternalGetLoadedAssemblies (MonoBoolean refonly, MonoError *error)
+{
+	error_init (error);
+	MonoDomain *domain = mono_domain_get ();
+	return get_assembly_array_from_domain (domain, refonly, error);
```

Comment:
I think we can specialize to `refonly == FALSE` for netcore. looks like `ReflectionOnlyGetAssemblies` just always returns an empty array.
https://github.com/mono/mono/blob/090dffb9a09f26dc250d8d09a46819d192290b0b/netcore/System.Private.CoreLib/shared/System/AppDomain.cs#L183

Reply:
I left the parameter and just pass false, but I can remove it entirely from the icall.

Reply:
I'd just get rid of it entirely.